src/gs-category-page.c

```diff
@@ -257,6 +276,8 @@ filter_selected (GtkListBox *filters, GtkListBoxRow *row, gpointer data)
 	if (row == NULL)
 		return;
 
+	//FIXME: use GDB and fatal warnings to track this down
+	// FIXME: this or something similar in this file assumes children are the content box but that's no longer true due to the addition of the USB box
```

Comment:
Use C-style comments. And use the `/* FIXME` consistently (notice one has a space before, the other has none).